tests/dom/global.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global window, document */
+
+import { window as globWindow, document as globDocument } from 'ckeditor5/utils/dom/global.js';
```

Comment:
I take back my previous comment. I didn't read this PR carefully before writing it. Now I have no idea why all this works at all.